pkg/apis/triggers/v1alpha1/event_listener_types.go

```diff
@@ -26,12 +27,25 @@ import (
 var _ apis.Validatable = (*EventListener)(nil)
 
 type EventListenerSpec struct {
-	TriggerBindingRefs []TriggerBindingRef `json:"triggerbindingrefs,omitempty"`
+	Triggers []Trigger `json:"triggers,omitempty"`
+}
+
+// +k8s:deepcopy-gen=true
+type Trigger struct {
+	TriggerBinding     TriggerBindingRef  `json:"triggerBinding"`
+	TriggerTemplate    TriggerTemplateRef `json:"triggerTemplate"`
+	ServiceAccountName string             `json:"serviceAccountName"`
```

Comment:
can we have docstrings for all the structs + attributes we're adding?

(looks like we need to turn on more linting maybe, there should be linting that catches when exported values don't have docstrings :thinking: )

Reply:
Adding `-E golint` to our lint command (`golangci-lint run`) _should_ allow us to get [this behavior](https://github.com/golang/lint/blob/master/lint.go), but when I tried it only complained about naming stutter errors. Maybe someone else wants to l

Reply:
kk! Ive opened https://github.com/tektoncd/plumbing/issues/64 in the plumbing repo so hopefully we can get this for both projects :D (and cli too?)

Reply:
For now, I have added docstrings for the existing public types.